### PR TITLE
feat: use hashRouter

### PR DESCRIPTION
**Pull request by marvinIsSacul**
fix: addAutoRewriteRules

### CODE REVIEW

File: packages/root-config/src/microfrontend-layout.html [hunk 1]
```diff
@@ -1,38 +1,23 @@
...
+        <route path="#!/groups"> </route>
+        <route path="#!/mediators"> </route>
+        <route path="#!/certificates"> </route>
+        <route path="#!/export-import"> </route>
+        <route path="#!/logs"> </route>
+        <route path="#!/about"> </route>
+        <route path="#!/profile"> </route>
+        <route path="#!/portal-admin"> </route>
+        <div style="flex-grow: 1">
+          <application name="@jembi/legacy-console"></application>
```
Comment (drizzentic) [inline packages/root-config/src/microfrontend-layout.html:20]: which formatter are you using?

**Review by drizzentic — APPROVED** [review]
LGTM! Just some minor formatting issues

File: packages/root-config/src/microfrontend-layout.html (hunk 1 above)
Comment (marvinIsSacul) [reply packages/root-config/src/microfrontend-layout.html:20]: I'm using prettier

Comment (marvinIsSacul) [reply packages/root-config/src/microfrontend-layout.html:20]: Should I undo them ?

**Pull request merged**